dockerfiles/builder.Dockerfile

```diff
@@ -0,0 +1,25 @@
+# A base container to build datamon and run CI jobs
+#
+# This image is updated every week on our CI
```

Comment:
can there be a reference dockerfile responsible for the image  and the repo that contains it here, or for infosec reasons, would such a link better be added to Confluence somewhere?

one way or another, what i'm missing here is why this comment is just saying "such an image exists" rather than "such an image exists, and this is where it came from."

Reply:
@ransomw1c that is a very good idea. Extracting this as a separate issue. I will proceed with writing more documentation, specifically for the docker part.

Reply:
@ransomw1c done. See PR #435